### ISSUE TITLE
Show priority of watching show in Plan to Wartch

### ISSUE THREAD

**Issue by anjandev** [z411/trackma#141]
Mal has an option to select your priority for shows in your plan to watch list section. It would be nice if trackma could show me this in the gtk/qt guis.

**Comment by z411** [comment]
I wasn't aware of this. Implementing this might be useful because VNDB also has a similar Wishlist with priorities. Do you have any suggestions on how it could be implemented in the user interfaces?

**Comment by anjandev** [comment]
I would like to sort my shows in the plan to watch by priority. It would be nice to have a seperate column for priority that only appears when I click the Plan to Watch tab.

**Comment by anjandev** [comment]
Similarily, it would nice to have more options that I could edit on mal from trackma. Suchas, this is mal when I click edit from their site:

![2015-09-04-182404_3600x1080_scrot](https://cloud.githubusercontent.com/assets/8602360/9697017/4190d776-5332-11e5-9ccd-bf439fa62eb2.png)

Things like start date and end date could be automated to be pushed. Suchas when I start an anime, trackma could automatically send mal my start date. Also, since trackma keeps track of how many episodes have been downloaded it could update these as well.

It also would be nice to have columns within the guis that can be enabled/disabled for these extra attributes.

**Comment by z411** [comment]
Yeah, I think it's time to make all these options visible and modifiable. I'm just not sure of two things:

1. How to implement them in the user interfaces; one idea is to add them in the Show details window when you double click the show, like Taiga does. Another idea is to add a Show detailed options button in the main window.
2. More importantly, not all sites have the exact same options. We have to find a way to make it cross-compatible between different sites (AniDB, MAL, VNDB, Hummingbird).

>Things like start date and end date could be automated to be pushed. Suchas when I start an anime, trackma could automatically send mal my start date.
Trackma already does this internally, it's just not shown anywhere. The settings dialog of the Qt interface has the option "Change start and finish dates automatically" in the Sync tab, it should be enabled by default.

**Comment by anjandev** [comment]
>1. How to implement them in the user interfaces; one idea is to add them in the Show details window when you double click the show, like Taiga does. Another idea is to add a Show detailed options expand arrow in the main window.

I think doing how taiga does it is prefect. Trackma could have two tabs when the new box opens. One for info and the other for detailed options. In the detailed options we could add all the options that taiga has such as manually defining a folder for the anime.

>2. More importantly, not all sites have the exact same options. We have to find a way to make it cross-compatible between different sites (AniDB, MAL, VNDB, Hummingbird).

Perhaps the engine can keep a database for all of these options and which sites support which options. The user is only able to change the option in the gui if their current site supports it. 

I could not find the option "Change start and finish dates automatically" in the Sync tab in the gtk interface.

**Comment by z411** [comment]
> I could not find the option "Change start and finish dates automatically" in the Sync tab in the gtk interface.

It's currently only in the Qt interface; the GTK settings dialog is still missing some options, I'll add them soon.

**Comment by Matoro** [comment]
So have there been any updates on this front?  I'm particularly interested in the tags part of this, as I maintain a pretty damn detailed tags system.  Back when I was on Windows I used MAL Updater 2, which has a tool which lists all entries along with an editable and sortable field for tags.  Something like that would be cool.

Also unrelated but probably small enough that it doesn't need its own issue is a command to issue from the cli to open a show's webpage:  most likely ```web show_number``` or something similar.  Should be pretty simple.

**Comment by Birdulon** [comment]
I could take a stab at tags on the Qt client.

**Comment by Matoro** [comment]
Oh, and for reference, I didn't think it would work which is why I haven't considered it up till now, but I just ran MAL Updater under Wine and it works fine--in fact there's even a dialog box that indicates that it detected a Linux system, not sure if it does anything though.  So I will be using that until this wonderful program adds that feature :)

**Comment by z411** [comment]
Are we sure APIs implement them? I'm not sure which one MAL implemented in update but not in fetching.
